components/engine_rocks/src/write_batch.rs

```diff
@@ -114,7 +92,7 @@ impl Mutable for RocksWriteBatch {
         self.wb.is_empty()
     }
 
-    fn should_write_to_engine(&self) -> bool {
+    fn should_write_opt(&self) -> bool {
```

Comment:
why call it `should_write_opt`?

Reply:
Yes. It does not has a options parameter.

Reply:
Oh, this was a sed mistake. Good catch.

Reply:
Fixed.